pkg/lint/load.go

```diff
@@ -283,6 +283,35 @@ func (cl *ContextLoader) filterDuplicatePackages(pkgs []*packages.Package) []*pa
 	return retPkgs
 }
 
+func (cl *ContextLoader) cleanSkipped(pkgs []*packages.Package) ([]*packages.Package, error) {
+	// Clean skipDirs from pkgs.
+	for _, skipDir := range cl.cfg.Run.SkipDirs {
+		for i, pkg := range pkgs {
+			if strings.HasSuffix(pkg.PkgPath, skipDir) {
+				pkgs = append(pkgs[:i], pkgs[i+1:]...)
+			}
+		}
+	}
+
+	// Clean skipFiles from pkgs.
+	filePath, err := os.Getwd()
+	if err != nil {
+		return nil, errors.New("failed to get wd")
+	}
```

Comment:
I suspect that this code block isn't compatible with `windows`. 

Because of this:

`skip-dirs` and `skip-files` work well with linux.
`skip-dirs` works will windows but `skip-files` doesn't.

I will research and find a full cross-os solution.

CC/ @ldez